components/container/edit-container.tsx

```diff
@@ -105,7 +105,7 @@ export const EditContainer = () => {
       <NoteNav />
       <DeleteAlert />
       <section className="h-full">
-        <MainEditor />
+        <MainEditor note={note} />
```

Comment:
Seems like typescript didn't catch this because of the async import. Didn't notice any side effects when adding this.